class7-week3/actions/markAsDone.js

```diff
@@ -0,0 +1,23 @@
+const Todo = require('../models/todo')
+const deserializeTodo = require('../util/deserializeTodo')
+
+module.exports = function markAsDone(request, response) {
+
+  const id = request.params.id
+  const todo = deserializeTodo(request, response)
```

Comment:
You don't need lines 7 and 8, and they will cause you errors here. What happens when you run this code in Postman?

Hint: That error you receive is coming from `../util/deserializeTodo` because there is no todo in the body of your HTTP request.